README.md

```diff
@@ -1,2 +1,2 @@
-# hello-world
-Getting started with hello-world
+# Hello World
```

Comment:
Usually, a contributor will come around and say: "I don't like the change on this line because it causes problems"
And contributors will come and argue their point.